spec.bs

```diff
@@ -410,6 +412,8 @@ items:
 :: A [=string=] or null
 : <dfn>filtering ID max bytes</dfn> (default: [=default filtering ID max bytes=])
 :: A positive integer
+: <dfn>requested max contributions</dfn> (default: null)
```

Comment:
Now that I've drafted up the calling APIs' spec changes, I'm debating renaming this to just "max contributions". I'm not sure "requested" really adds anything.

Reply:
Yeah, either works, but I maybe have a slight preference for dropping "renaming"